community.hachyderm.io/content/en/docs/moderation/covenant/_index.md

```diff
@@ -0,0 +1,126 @@
+---
+title: "Moderator Covenant"
+linkTitle: "Covenant"
+weight: 40
+description: >
+   How Hachyderm moderators moderate Hachyderm.
+---
+
+This is the set of principles that Hachyderm moderators agree to
+inform their decisions and judgment calls when
+creating and maintaining Hachyderm as a safe space and enforcing
+server rules. This is because first and foremost:
+
+> Hachyderm moderators acknowledge the importance of server rules /
+> Codes of Conduct that are complete and clear. Hachyderm moderators
+> _also_ acknowledge that the entirety of human behavior cannot be
+> captured by an itemized list, no matter how many subsections it has, and
+> therefore use the following principles to ensure that we are
+> _always_ able to take action even in situations where a reported
+> infraction “falls between the cracks”.
+
+1. **We will prioritize the vulnerable.**<br />All actions we take will prioritize the most vulnerable, full stop.
+1. **We acknowledge that we will make mistakes.**<br />
+We acknowledge that we are not infallible. We will constantly be
+learning and growing and will respond to our mistakes with
+acknowledgement, care, and do what is necessary to undo or
+mitigate the harm done.
+1. **We will moderate with respect.**<br />
+We will handle our communications with users and accounts that
+have been reported for moderation with respect.
+1. **We acknowledge and understand that we are strangers on this pale blue dot.**<br />
+The reality is that the vast majority of user interactions are
+between strangers, even if familiarity increases with time. This
+means that in almost all reported situations: the reported user is
+a stranger to us as moderators and that the reported user is a
+stranger to other users in the interaction (on a personal level,
+even if the account is recognizable). This means that while we
+will look at the reported user on the surface to try and
+understand possible intent, we acknowledge that it is not possible
+to use intent or presumed / guestimated intent alone to inform
+what moderation action(s), if any, to take.
+1. **We will prioritize impact over intent.**<br />
+Whenever we look into a reported interaction, we look at as much
+of the situation we can see. This means we do due diligence on
+seeing what, if any, factors are contributing to the situation and
+if that situation is escalating or at risk of escalating. Since we
+acknowledge that We Are Strangers, that means we are doing this
+based on an understanding of people, in general, and the
+intersectionalities at play. Also, again since We Are Strangers,
+we then use the impact of the reported user action to inform what
+moderation action(s) to take, if any.
```

Comment:
I think we can remove "Also, again since We Are Strangers,
we then use the impact of the reported user action to inform what
moderation action(s) to take, if any." 
and instead explain why it's important to prioritize impact:

"Regardless of intent or whether actions and words were purposeful, the targeted or affected person is still harmed. That’s why it is critical to prioritize impact and acknowledge the harm that was caused."